### PR TITLE
Update absolute hydration example

### PR DESCRIPTION
**Pull request by maxentile**
Resolves broken references, largely rewrites script to more closely resemble protocols used in `fe` module.

### CODE REVIEW

File: examples/ahfe.py [hunk 1]
```diff
@@ -4,142 +4,112 @@
...
 
-# 1. build water box
-# 2. build ligand
-# 3. convert to recipe
+from pathlib import Path
 
-# construct an RDKit molecule of aspirin
-# note: not using OpenFF Molecule because want to avoid the dependency (YTZ?)
-romol = Chem.AddHs(Chem.MolFromSmiles("CC(=O)OC1=CC=CC=C1C(=O)O"))
+from tqdm import tqdm
```
Comment (badisa) [inline examples/ahfe.py]: We do not currently have tqdm as a requirement in the repo, should we add it?

**Comment by badisa** [comment]
Naive question, how does this script differ from using `fe/model_rabfe.py::AbsoluteHydrationModel`?

File: examples/ahfe.py (hunk 1 above)
Comment (maxentile) [reply examples/ahfe.py]: Ahh, I misremembered! (I remembered [pinning](https://github.com/proteneer/timemachine/pull/442#discussion_r645881764) the version in an earlier PR but it was unmerged.)

Can add tqdm as a dependency, remove the progress bar (just use prints), or use a different progress bar

**Comment by maxentile** [comment]
>Naive question, how does this script differ from using fe/model_rabfe.py::AbsoluteHydrationModel?

Good question -- I need to look more carefully if parts of this should be replaced with fe/model_rabfe.py::AbsoluteHydrationModel or fe/model_rabfe.py::AbsoluteStandardHydrationModel .

Initially, I was trying to make the minimal set of changes to get `examples/ahfe.py` to run.

**Comment by badisa** [comment]
> > Naive question, how does this script differ from using fe/model_rabfe.py::AbsoluteHydrationModel?
> 
> Good question -- I need to look more carefully if parts of this should be replaced with fe/model_rabfe.py::AbsoluteHydrationModel or fe/model_rabfe.py::AbsoluteStandardHydrationModel .
> 
> Initially, I was trying to make the minimal set of changes to get `examples/ahfe.py` to run.

The changes as they are look fine, and I only wondered about the differing styles. Not sure this example is worth too much more attention, and we can decide at a. later date which style is preferred.

File: examples/ahfe.py (hunk 1 above)
Comment (maxentile) [reply examples/ahfe.py]: Removed in https://github.com/proteneer/timemachine/pull/580/commits/297fdec7509bd14e4175a211a14b6ca3be6964ec

**Comment by maxentile** [comment]
Closing, probably better to re-write using `fe/model_rabfe.py::AbsoluteStandardHydrationModel`

**Pull request closed without merging**